includes/class-comment.php

```diff
@@ -520,14 +542,17 @@ public static function get_comment_types() {
 	/**
 	 * Is this a registered comment type.
 	 *
-	 * @param string $slug The name of the type.
+	 * @param string $name The name of the type.
```

Comment:
I'd leave it as slug, as that's what it is (hence my proposal to rename `get_comment_type_names()` to `get_comment_type_slugs()`). The name would be the translated "Repost" and "Like".

Reply:
so reverting this commit?